contracts/lido_terra_hub/src/testing/mock_querier.rs

```diff
@@ -219,6 +255,10 @@ impl WasmMockQuerier {
                         airdrop_registry_contract: Some(
                             api.addr_canonicalize(&String::from("airdrop")).unwrap(),
                         ),
+                        airdrop_withdrawal_account: Some(
+                            api.addr_canonicalize(&String::from("airdrop_withdrawal_account"))
```

Comment:
Why do we do `&String::from("")` when we can do just `""`?

Reply:
sure, we can.